.github/workflows/publish.yml

```diff
@@ -0,0 +1,94 @@
+# Publishes built jars to distribution platforms
+name: Publish
+
+on:
+  push:
+    tags:
+      - 'v*.*.*' # any semver tag, e.g. v1.2.3
```

Comment:
Is this regex? As 'v..' would be valid release tag

Reply:
I have updated this to require numbers in those positions instead. It's not quite regex, but a glob-like matcher as per the [docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet).